### PR TITLE
Remove obsolete dependencies

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
We're sitting at >6MB of total bundle size right now. This PR is aimed at removing unused dependencies for the next release

**Candidates:**
- [x] `wicked-good-xpath`: Originally brought in to cover up lacking xpath capabilities of IE, can go now since we dont' support it
- [X] `lodash`: Can be dropped in favor of `lodash-es` which we're already using. ~Only a few `.chain` methods need some brainpower, the rest is just a bit of busywork~ Update: Was straightfoward to remove and CI was happy. `lodash` is still being used in the `tests/` but that's less of a pressing matter
- [x] `moment`: Unmaintained AFAIK and we're already using `luxon` for dates and times
- [X] `inert-polyfill`: Unused library that helped with focus management
- [ ] ...tbd...

Also got this hint when building locally, might be worthwile to take a look https://evilmartians.com/chronicles/postcss-8-plugin-migration

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong>
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by pascalwengerter** [comment]
Also got this hint when building locally, might be worthwile to take a look https://evilmartians.com/chronicles/postcss-8-plugin-migration

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16861/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10CreateDelete</strong> https://drone.owncloud.com/owncloud/web/16861/7/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:59

**Comment by pascalwengerter** [comment]
Getting close to where I wanted here. If we can fix #5342 and tick the checkboxes in the PR description then bundle size is down to ~4.5MB (still a lot but reduced by 25%)

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16871/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroups</strong> https://drone.owncloud.com/owncloud/web/16871/20/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroups-shareWithGroups-feature-277.png
  

### webUISharingInternalGroups-shareWithGroups-feature-308.png
  

### webUISharingInternalGroups-shareWithGroups-feature-309.png
  

### webUISharingInternalGroups-shareWithGroups-feature-333.png
  

### webUISharingInternalGroups-shareWithGroups-feature-334.png
  

### webUISharingInternalGroupsEdgeCases-shareWithGroupsEdgeCases-feature-41.png
  

### webUISharingInternalGroupsEdgeCases-shareWithGroupsEdgeCases-feature-42.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsToRoot</strong> https://drone.owncloud.com/owncloud/web/16871/21/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroupsToRoot-shareWithGroups-feature-249.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-277.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-278.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-299.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-300.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-36.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-37.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersExpire</strong> https://drone.owncloud.com/owncloud/web/16871/25/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-104.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-105.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-179.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-180.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-207.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-208.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-211.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-24.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-240.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-54.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-84.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-85.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-171.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-172.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-199.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-200.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-203.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-21.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-232.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-48.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-76.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-77.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-96.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-97.png
  


</p></details>

File: packages/web-app-files/src/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.vue [hunk 1]
```diff
@@ -243,14 +243,12 @@ export default {
   },
 
   mounted() {
+    console.log(this.enteredExpirationDate, this.expirationDate)
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.vue:246]: This needs to go

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16997/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsToRoot</strong> https://drone.owncloud.com/owncloud/web/16997/21/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroupsToRoot-shareWithGroups-feature-249.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-277.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-278.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-299.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-300.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-36.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-37.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroups</strong> https://drone.owncloud.com/owncloud/web/16997/20/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroups-shareWithGroups-feature-277.png
  

### webUISharingInternalGroups-shareWithGroups-feature-308.png
  

### webUISharingInternalGroups-shareWithGroups-feature-309.png
  

### webUISharingInternalGroups-shareWithGroups-feature-333.png
  

### webUISharingInternalGroups-shareWithGroups-feature-334.png
  

### webUISharingInternalGroupsEdgeCases-shareWithGroupsEdgeCases-feature-41.png
  

### webUISharingInternalGroupsEdgeCases-shareWithGroupsEdgeCases-feature-42.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersExpire</strong> https://drone.owncloud.com/owncloud/web/16997/25/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-104.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-105.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-179.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-180.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-207.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-208.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-211.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-24.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-240.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-54.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-84.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-85.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-171.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-172.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-199.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-200.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-203.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-21.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-232.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-48.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-76.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-77.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-96.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-97.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsToRoot</strong> https://drone.owncloud.com/owncloud/web/17033/21/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroupsToRoot-shareWithGroups-feature-299.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-300.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-36.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-37.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17038/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsToRoot</strong> https://drone.owncloud.com/owncloud/web/17038/21/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroupsToRoot-shareWithGroups-feature-299.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-300.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-36.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-37.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17047/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsToRoot</strong> https://drone.owncloud.com/owncloud/web/17047/21/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroupsToRoot-shareWithGroups-feature-299.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-300.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-36.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-37.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17063/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersExpire</strong> https://drone.owncloud.com/owncloud/web/17063/25/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-211.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-240.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-203.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-232.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISTrashbinUploadMoveJourney</strong> https://drone.owncloud.com/owncloud/web/17064/60/1
The following scenarios passed on retry:
- webUITrashbinDelete/trashbinDelete.feature:112

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17064/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersExpire</strong> https://drone.owncloud.com/owncloud/web/17064/25/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-240.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-232.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by kulmann — APPROVED** [review]
Looks good to me! 💪

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17067/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic</strong> https://drone.owncloud.com/owncloud/web/17067/6/1
The following scenarios passed on retry:
- webUIPreview/imageMediaViewer.feature:35

**Pull request merged**